db_client/models/dfce/family.py

```diff
@@ -229,6 +230,8 @@ class FamilyDocument(Base):
         lazy="joined",
     )
 
+    metadata_value = sa.Column(postgresql.JSONB)
```

Comment:
Is the name here meant to match the migration?

Reply:
Absolutely is - what a long day 🫠